jax/_src/core.py

```diff
@@ -2117,7 +2125,7 @@ def _pvary_abstract_eval(*args, axes, axis_index_groups):
   if not config._check_vma.value:
     return args
   assert isinstance(axes, tuple)
-  arg_vma = [a.vma for a in args]
+  arg_vma = [a.vma for a in args if isinstance(a, ShapedArray)]
```

Comment:
Do we currently need this line, given the change to standard_insert_pvary below? That is, it seems nicer to ensure that we don't call pvary on non-ShapedArray-typed inputs in the first place.

Reply:
oh you're absolutely right! We don't need it anymore. I've removed this diff